### DISCUSSION TITLE
Missing or currupt files

### DISCUSSION THREAD

**Discussion by rohitchormale** [seaweedfs/seaweedfs#4670] [Q&A]
I have a bare git repo on seaweedfs storage. After some initial commits, somehow repo is getting currupted. If I run 'git fsck', I can see multiple git object files are missing/currupt. 

- Interestingly the storage is mounted in 2 other pods where those files are readable. If I mount storage at more locations, files are present. So volume looks good. 
- On currupt mount, I can create/update/read/delete new files, so entire mount is not failed. 
- The fuse mount shows the files with size but  can't read them. 

 Mount is done as pvc with help of csi driver. Further debugging shows that this is happening with static pvc only. Dynamic pvc is good. What could cause such issue ? Tested with Seaweedfs version is 3.44 and 3.53

**Comment by rohitchormale** [comment]
Looks like after v3.54 issue is not there any more. I will do more testing soon and will share findings.

**Comment by rohitchormale** [comment]
Looks like there is some kind of regression in 3.55. We are facing file curruption issue with version 3.55 but not with 3.54.

@chrislusf

**Reply by chrislusf** [reply]
Is it with "weed mount " and large files?

**Reply by rohitchormale** [reply]
These are git object files, small in size. It could be related with "weed mount" as that is what used by csi driver.
